eslint.config.js

```diff
@@ -19,6 +19,8 @@ module.exports = [
       '!test/test-cases/*/*.test.js',
       'fixtures/*/*',
       '**/@loadable/**/*',
+      '**/*.test.js',
+      '**/*.test.ts',
```

Comment:
This was undone in the previous PR. Was it intentionally changed back in this PR?

Reply:
This was a merge issue. I'm rolling it back.